react-components/src/components/RevealToolbar/SetFlexibleControlsType.tsx

```diff
@@ -189,13 +189,15 @@ function getIcon(controlsType: FlexibleControlsType): IconType {
   }
 }
 
-function asFlexibleCameraManager(manager: CameraManager): IFlexibleCameraManager | undefined {
-  // instanceof don't work within React, so using safeguarding
-  const flexibleCameraManager = manager as IFlexibleCameraManager;
-  return flexibleCameraManager.controlsType === undefined ? undefined : flexibleCameraManager;
-}
-
 const ButtonsContainer = styled.div`
   display: flex;
   flex-direction: column;
 `;
+
+export function asFlexibleCameraManager(
+  manager: CameraManager
+): IFlexibleCameraManager | undefined {
+  // instanceof don't work within React, so using safeguarding
+  const flexibleCameraManager = manager as IFlexibleCameraManager;
+  return flexibleCameraManager.controlsType === undefined ? undefined : flexibleCameraManager;
+}
```

Comment:
You can create a user-defined type guard instead: https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

This will have slightly nicer semantics

Reply:
I think this is fine - but you can have a look at Mr. greenbech's link if you want

Reply:
@greenbech It is what you suggested. I have used the like to create this code